nfs-subdir-external-provisioner.yaml

```diff
@@ -21,19 +21,14 @@ pipeline:
       tag: nfs-subdir-external-provisioner-${{package.version}}
       expected-commit: c2a2d5d544781e3d3e4d7a7e2d21a8e64cf6d9d1
 
-  # I added `go mod edit -go=1.17` because when I do `go get golang.org/x/net@v0.7.0,`
-  # to fix a High CVE it gave me an error unsafe.Slice requires go1.17 or later (-lang was set to go1.16; check go.mod.
+  - uses: go/bump
+    with:
+      deps: github.com/prometheus/client_golang@v1.11.1 golang.org/x/text@v0.9.0 golang.org/x/net@v0.17.0 gopkg.in/yaml.v3@v3.0.0-20220521103104-8f96da9f5d5e
```

Comment:
Minor UX feedback, I think we should standardize on this for better readability/diffs

```
deps: >-
  github.com/prometheus/client_golang@v1.11.1
  golang.org/x/text@v0.9.0 golang.org/x/net@v0.17.0
  gopkg.in/yaml.v3@v3.0.0-20220521103104-8f96da9f5d5e
```